src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+/**
+ * Encapsulate the command that allow users to add new task to the TaskList,
+ * which is-a Command.
+ */
+public class AddCommand extends Command {
+
+    private Task newTask;
```

Comment:
```suggestion
    private final Task newTask;
```
Suggestion for the addition of `final` modifier.

Reply:
I did not implement it as final as I may need to modify the said variable later on.